src/contracts/pods/EigenPod.sol

```diff
@@ -248,16 +261,17 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     ) external onlyWhenNotPaused(PAUSED_EIGENPODS_VERIFY_OVERCOMMITTED) {
        // ensure that the blockNumber being proven against is not "too stale", i.e. that the validator was *recently* overcommitted.
         require(oracleBlockNumber + VERIFY_OVERCOMMITTED_WINDOW_BLOCKS >= block.number,
-            "EigenPod.verifyOvercommittedStake: specified blockNumber is too far in past");
+            "EigenPod.verifyBalanceUpdate: specified blockNumber is too far in past");
 
-        require(validatorStatus[validatorIndex] == VALIDATOR_STATUS.ACTIVE, "EigenPod.verifyOvercommittedStake: Validator not active");
+        bytes32 validatorPubkeyHash = validatorFields[BeaconChainProofs.VALIDATOR_PUBKEY_INDEX];
 
+        {
+            VALIDATOR_STATUS validatorStatus = _validatorPubkeyHashToInfo[validatorPubkeyHash].status;
+            require(validatorStatus == VALIDATOR_STATUS.ACTIVE || validatorStatus == VALIDATOR_STATUS.OVERCOMMITTED, "EigenPod.verifyBalanceUpdate: Validator not active or overcommitted");
```

Comment:
there should be no such thing as `VALIDATOR_STATUS.OVERCOMMITTED`?

Reply:
oh yeah good point since the shares are always up to date.